source/braille.py

```diff
@@ -2755,6 +2788,8 @@ def initialize():
 	if newDriverName:
 		config.conf["braille"]["display"] = newDriverName
 	handler.setDisplayByName(config.conf["braille"]["display"])
+	pre_speech.register(_showSpeechInBraille)

```

Comment:
```suggestion
	# noqa: F401 avoid module level import to prevent cyclical dependency
	# between speech and braille
	from speech.extensions import pre_speech, pre_speechCanceled
	pre_speech.register(_showSpeechInBraille)
```